pkg/validator/types.go

```diff
@@ -81,4 +81,7 @@ const (
 	invalidSrcURL   = "Unable to access the specified URL(\"%s\") in file injection spec for the role (%s). error: %s."
 
 	maxMemberLimit = "Maximum number of total members per KD cluster supported is %d."
+
+	nameLimit       = "When using the ClusterRole naming scheme, the total length of KD cluster name + any of its role IDs must not exceed 56 characters."
+	badNamingScheme = "Naming scheme not valid in the config file."
```

Comment:
Can get rid of badNamingScheme if we remove the check for it as per the other comment.